syntax = "proto3";

package com.android.fastdeploy;

option java_package = "com.android.fastdeploy";
option java_outer_classname = "ApkEntryProto";
option java_multiple_files = true;

message APKDump {
    string name = 1;
    bytes cd = 2;
    bytes signature = 3;
    string absolute_path = 4;
}

message APKEntry {
    bytes md5 = 1;
    int64 dataOffset = 2;
    int64 dataSize = 3;
}

message APKMetaData {
    string absolute_path = 1;
    repeated APKEntry entries = 2;
}
